test/common/index.js

```diff
@@ -24,7 +24,7 @@
 const process = global.process;  // Some tests tamper with the process global.
 
 const assert = require('assert');
-const { exec, execSync, spawnSync } = require('child_process');
+const { exec, execSync, spawnSync, spawn: spawnAsync } = require('child_process');
```

Comment:
```suggestion
const { exec, execSync, spawnSync, spawn } = require('child_process');
```

I don’t think you have a conflict with naming it `spawn`, now, do you? The name `spawnAsync` makes it seem like it’s an async function/returns a promise.

Reply:
Ah, true, not any more 👍